jib-gradle-plugin/README.md

```diff
@@ -108,6 +108,21 @@ eval $(minikube docker-env)
 gradle jibDockerBuild
 ```
 
+Alternatively, the environment can be configured in the task with two optional parameters:
```

Comment:
We don't update `README` until we actually make a release, because it may confuse the users of the current latest version. Should be removed here. It'd be nice to put up a separate PR.

Reply:
will do